docs/docs/command-line-flags.md

```diff
@@ -47,6 +47,7 @@ The core functionality flags can be also set by environment variable `MARATHON_O
     - "vips" can be used to enable the networking VIP integration UI.
     - "task\_killing" can be used to enable the TASK\_KILLING state in Mesos (0.28 or later)
     - "external\_volumes" can be used if the cluster is configured to use external volumes.
+    - "mesos_healthchecks" can be used if ti enable the Mesos HealthChecks (Mesos 1.1 or later).
```

Comment:
This blog post is partially correct. Mesos Health Checks were introduced as an **experimental** feature in [Mesos 1.1.0](https://github.com/apache/mesos/blob/1.1.0/CHANGELOG#L15-L30)